src/web/modules/gateway_accounts/detail.njk

```diff
@@ -433,6 +433,34 @@
     </div>
   {% endif %}
 
+    {% if account.recurring_enabled === false %}
+    <div>
+      <h1 class="govuk-heading-s">Recurring card payments disabled</h1>
+      <p class="govuk-body">Allows the account to take recurring card payments.</p>
+      <form method="POST" action="/gateway_accounts/{{ gatewayAccountId }}/toggle_recurring_enabled">
+        {{ govukButton({
+          text: "Enable recurring card payments",
+          classes: "govuk-button--warning"
+          })
+        }}
+        <input type="hidden" name="_csrf" value="{{ csrf }}">
+      </form>
+    </div>
+  {% endif %}
+
+  {% if account.recurring_enabled === true %}
+    <div>
+      <h1 class="govuk-heading-s">What’s recurring?</h1>
```

Comment:
As much as I appreciate Gavin-and-Stacey-driven-development I think it would be a good idea for the description for what constitutes "recurring payments" to be consistent whether it's enabled or not. 

**Recommendation:** 

h1 Recurring card payments disabled
p Allows the account to take recurring card payments. Shows agreements and their payment instruments in the admin tool.

h1 Recurring card payments enabled
p Allows the account to take recurring card payments. Shows agreements and their payment instruments in the admin tool.

There may be a point in the future where the amount of functionality tied to the flag changes (i.e we show agreements by default to encourage services to discover the feature) but I think it's worth being more specific now at risk of it being out of date for short periods

Reply:
You’re no fun! I updated the text.